slider/beatmap.py

```diff
@@ -909,6 +909,43 @@ def _get_as_bool(groups, section, field, default=no_default):
         )
 
 
+def _moving_average_by_time(data, delta, num):
```

Comment:
this is often called downsampling

Reply:
I'm not entirely sure this is downsampling; wouldn't that imply that what I'm generating is somehow the original signal had it been sampled less frequently? The original signal in this case isn't sampled at all, it is the exact data. I'm adding an artificial smoothing that didn't exist before. Still I'm not that familiar with these techniques so I might be wrong :P